style.css

```diff
@@ -1,6 +1,606 @@
+  align-items: center;
+}
+
+.navs {
+  display: flex;
+  width: 60%;
+  min-width: 500px;
+}
+
+.tools a { color: #f7f7f7; }
+
+.header { padding: 20px 0; }
+
+#headline {
+  justify-content: space-around;
+  background-image: url('images/mobBgBHome.png');
+  background-position: bottom center;
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.progs,
+.container-main {
+  display: flex;
+  margin: 0 auto;
+  min-width: 280px;
+  width: calc(100vw - 40px);
+  max-width: 700px;
+  flex-wrap: wrap;
+}
+
+#headline > .container-main { margin: 30px 0; }
+
+#abt-main-desc,
+#main-desc {
+  position: relative;
+  border: 2px solid #fff;
+  padding: 20px;
+  margin: 20px 0;
+  box-shadow: 0 3px 10px rgb(0 0 0 / 20%);
+}
+
+.cards {
+  min-height: 130px;
+  min-width: 300px;
+  display: flex;
+  justify-content: flex-start;
+  gap: 10px;
+  width: 100%;
+  max-width: 700px;
+}
+
+#cards-container {
+  row-gap: 20px;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 30px 0;
+}
+
+.img-ph {
+  width: 135px;
+  max-height: 130px;
+  background-image: url('images/angCB.png');
+  background-position: top left;
+  background-repeat: no-repeat;
+  flex: 0;
+}
+
+.card-img {
+  height: 100px;
+  width: 100px;
+  margin: 15px;
+}
+
+.card-details {
+  display: flex;
+  flex-direction: column;
+  min-width: 200px;
+  justify-content: space-evenly;
+  max-width: calc(100% - 130px);
+}
+
+.prog-img-ph {
+  width: 32px;
+}
+
+.prog-img {
+  width: 100%;
+}
+
+.topic {
+  width: 90px;
+}
+
+#feature-speaker {
+  background-color: #fff;
+  padding: 30px 20px;
+}
+
+/* ||  ====== Patterns ======= || */
+
+.pattern-2 {
+  position: absolute;
+  background-image: url('images/capsPat.png');
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 60px;
+  height: 60px;
+  background-size: cover;
+}
+
+#desc-pat-tr {
+  top: -62px;
+  right: 20px;
+  background-position: center 20px;
+}
+
+hr {
+  width: 25px;
+  background-color: #ec5242;
+  border: none;
+  height: 2px;
+  margin-top: 7px;
+}
+
+/* || ===== Main Program ======  */
+
+#programs {
+  background-image: url(images/pat-m.png);
+  background-repeat: repeat;
+  background-color: #272a31fc;
+  color: #d3d3d3;
+  justify-content: space-around;
+  padding: 40px 0;
+}
+
+.progs {
+  row-gap: 10px;
+  margin: 30px 0;
+}
+
+.pro-item {
+  background-color: rgba(255, 255, 255, 0.05);
+  padding: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+}
+
+#abt-partners-ph,
+#partners-ph {
+  background-color: #272a31;
+  padding: 20px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.partners-list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  gap: 20px;
+  margin-top: 18px;
+  row-gap: 10px;
+}
+
+.partners {
+  min-width: 60px;
+  width: 26%;
+  justify-content: center;
+}
+
+#footer { background-color: #fff; }
```

Comment:
Please be sure that your Partners and footer section covers 100% of the width of the screen 🤓

![image](https://user-images.githubusercontent.com/15807343/170843605-07598d4c-ad40-45a5-884d-8da8fdee68bc.png)